--- internal/native/BUILD.bazel
+++ internal/native/BUILD.bazel
@@ -14,6 +14,7 @@ go_library(
         "verifier.go",
     ],
     cgo = True,
+    cdeps = [":pact_ffi_cc_import"],
     clinkopts = select({
         "@io_bazel_rules_go//go/platform:android_amd64": [
             "-L/tmp -L/opt/pact/lib -L/usr/local/lib -Wl,-rpath -Wl,/opt/pact/lib -Wl,-rpath -Wl,/tmp -Wl,-rpath -Wl,/usr/local/lib -lpact_ffi",
@@ -81,3 +82,9 @@ go_test(
         "@org_golang_google_protobuf//proto",
     ],
 )
+
+cc_import(
+    name = "pact_ffi_cc_import",
+    shared_library = "@@//rules_pact:pact_ffi",
+    visibility = ["//visibility:public"],
+)
\ No newline at end of file
